docs/UserGuide.md

```diff
@@ -165,6 +169,10 @@ PropertyWhiz's data are saved as a JSON file `[JAR file location]/data/propertyw
 If your changes to the data file makes its format invalid, PropertyWhiz will discard all data and start with an empty data file at the next run.
 </div>
 
+### Multi-command input
+
+You may copy and paste multiple lines of commands into to the command box. Press the Enter key to execute the first command and autofill the next one. This continues until you enter a different command or all commands have been executed.
```

Comment:
Typo: replace `into to` with `into`
Suggestion: Append `autofill the next one` with ` in the command box`, i.e. `autofill the next one in the command box` to be more explicit.